Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM alpine:3.2
+MAINTAINER dengnan@google.com vmarmol@google.com vishnuk@google.com jimmidyson@gmail.com
+
+WORKDIR /gopath/src/github.com/google/cadvisor
+COPY    . /gopath/src/github.com/google/cadvisor
+
+RUN apk add --update --repository=http://dl-4.alpinelinux.org/alpine/edge/community/ -t build-deps go sqlite-dev btrfs-progs-dev bash linux-headers make git gcc g++ && \
```

Comment:
yeah, the docker build isn't optimized to support small image sizes.  I really don't have enough context on how the cadvisor image is used to know if the increase in size is an issue for folks using it.  I like your approach because it provides a consistent way of building cadvisor, and having docker hub build the image sounds like it could reduce maintenance cost.